test/test_topography.py

```diff
@@ -96,9 +96,9 @@ def test_attribute_error(self):
         t = UniformLineScan(h, 4)
 
         with self.assertRaises(AttributeError):
-            t.scale_factor
+            t.heights_scale_factor
```

Comment:
Is here a typo? `t.height_scale_factor` without `s`?

Reply:
Good catch, fixed this